packages/support/package.json

```diff
@@ -88,7 +87,6 @@
     "pngjs": "7.0.0",
     "read-pkg": "5.2.0",
     "resolve-from": "5.0.0",
-    "rimraf": "3.0.2",
```

Comment:
I can observe rimraf is used in at least 3 build scripts. They will stop working if we just remove it complete. Either include it into devDeps or update these build scripts where it is used

Reply:
let me do a more thorough check and get back on this. I assumed test passing would be enough. Thanks for being patient

Reply:
rimraf is a devDependency in the main package.json [here](https://github.com/appium/appium/blob/e7df6fd4f054d6ad6eb4125f77e7e0d58bc05ebf/package.json#L134).
Also, the below instances are using `npx`, AFAIK, we don't need it as a dependency right when we run directly via using `npx` ( assuming this is correct, we might be able to remove `rimraf` as a devDepenency too right? unless we want to keep it to ensure installation of rimraf even via npx during execution doesn't cause an issue )
[1](https://github.com/appium/appium/blob/e7df6fd4f054d6ad6eb4125f77e7e0d58bc05ebf/package.json#L37), [2](https://github.com/appium/appium/blob/e7df6fd4f054d6ad6eb4125f77e7e0d58bc05ebf/packages/fake-driver/package.json#L42), [3](https://github.com/appium/appium/blob/e7df6fd4f054d6ad6eb4125f77e7e0d58bc05ebf/packages/typedoc-plugin-appium/package.json#L27)

If these aren't the instances you were talking about could you point me to one which you are talking about?
The rest of the instances that I could find using `ag` on my local machine of `rimraf` seem to be using `fs.rimraf` or `fs.rimrafSync` which we have defined.

Reply:
@boneskull can you take a look?

Reply:
correct, we can keep instances of `npx rimraf` and those will not require it to be in devDeps.

Reply:
@mykola-mokhnach anything pending on me? or we are just waiting for a review from @boneskull.